tests/functional/firefox/test_win10_welcome.py

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.win10_welcome import Win10WelcomePage
+
+
+@pytest.mark.skip_if_firefox
```

Comment:
The test would fail in Firefox depending on whether or not it was the default browser on the machine. It would also rely on UITour configs being set correctly. Will add comment as suggested!